changelog/7991.bugfix.rst

```diff
@@ -0,0 +1 @@
+Added support within `~.parse_time` for the timestamp ``%Y%m%d%H%M``.
```

Comment:
```suggestion
Added support within `~.parse_time` for the timestamp ``%Y%m%d%H%M`` and ``%Y%m%d_%H%M``.
```
I forgot about the other one.

Reply:
I have also added the ``%Y%m%dT%H%M`` in changelog file